path_config_test.go

```diff
@@ -535,6 +535,62 @@ func TestConfig_LocalJWTRenewal(t *testing.T) {
 	}
 }
 
+func TestConfig_SystemCaJWT(t *testing.T) {
+	testCases := map[string]struct {
+		config              map[string]interface{}
+		setupInClusterFiles bool
+		expected            *kubeConfig
+	}{
+		"no CA or JWT, default to system": {
+			config: map[string]interface{}{
+				"kubernetes_host": "host",
+			},
+			setupInClusterFiles: true,
+			expected: &kubeConfig{
+				PublicKeys:           []interface{}{},
+				PEMKeys:              []string{},
+				Host:                 "host",
+				CACert:               testLocalCACert,
+				TokenReviewerJWT:     testLocalJWT,
+				DisableISSValidation: true,
+				DisableLocalCAJwt:    true,
+			},
```

Comment:
It looks like this test is failing; is the expected kubeConfig correct here?

Reply:
Updated in [a0dfafe](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/150/commits/a0dfafee6502d974689c2ce60a30265583d37d1f) to test the right code path. The `localCACert` is added to the OS trust store however we can remove this entirely and generate our own `self-signed cert` and add it to the OS trust sore when a cert is not supplied.